daily_tasks/commands/main_commands.py

```diff
@@ -1,20 +1,25 @@
 """Main commands of this CLI"""
 import json
+
 import click as ck
 from daily_tasks.commands import utilities
 
 
 @ck.command
-def create_tasks_file() -> None:
+@ck.option('-f', '--task_file_name',
```

Comment:
Change --task_file_name to --file_name, otherwise Click module can't pass the value given to this option to the function's parameter file_name.